xwiki-platform-core/xwiki-platform-refactoring/xwiki-platform-refactoring-api/src/main/java/org/xwiki/refactoring/internal/job/CreateJob.java

```diff
@@ -158,4 +161,27 @@ private void maybeCreate(DocumentReference newDocumentReference, DocumentReferen
             this.modelBridge.copy(templateDocumentReference, newDocumentReference, this.request.getUserReference());
         }
     }
+
+    /*
+     * TODO: Refactor since the same code is used in {@link MoveJob}.
```

Comment:
Indeed. We could put in in a separate component that is injected into the Create and Move jobs (I think I prefer this instead of moving the code in a base class).